packages/web3-hooks/base/src/useReverseAddress.ts

```diff
@@ -14,12 +14,7 @@ export function useReverseAddress<T extends NetworkPluginID>(
     })
     const { NameService, Others } = useWeb3State(pluginID)
 
-    const isBad =
-        !Others?.chainResolver.isValid(chainId) ||
-        !address ||
-        !Others.isValidAddress?.(address) ||
-        Others.isZeroAddress?.(address) ||
-        !NameService
+    const isBad = !address || !Others?.isValidAddress?.(address) || Others.isZeroAddress?.(address) || !NameService
```

Comment:
We can remove `chainId` parameter from `NameService.reverse` as well, since it's unused now.